cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaClusterRebalanceAssemblyOperator.java

```diff
@@ -321,12 +322,12 @@ public void onClose(KubernetesClientException e) {
                                     return Future.succeededFuture();
                                 }
                             }, exception -> {
-                                    log.error("{}: Status updated to [NotReady] due to error", exception);
+                                    log.error("{}: Status updated to [NotReady] due to error", exception.getMessage());
```

Comment:
I think that it was wrong before but even now :-(
The purpose is not printing the exception message instead of the `{ }` placeholder. That placeholder is there for printing the `reconciliation` information so the right line should be.

```java
log.error("{}: Status updated to [NotReady] due to error", reconciliation, exception);
```